lib/assets/tree/main.css

```diff
@@ -0,0 +1,38 @@
+.app {
+  font-family: "JetBrains Mono", monospace;
+  font-size: 0.875rem;
+}
+
+div.item {
+  display: flex;
+}
+
+div.clickable {
+  cursor: pointer;
+}
+
+.ri {
+  font-size: 1.25rem;
+  vertical-align: middle;
+}
+
+.ri-empty {
+  display: inline-block;
+  width: 1.25rem;
+}
+
+ol {
+  display: none;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  padding-left: 2rem;
+}
+
+ol.show {
+  display: block;
+}
+
+div.suffix {
+  margin-left: 1.25rem;
+}
```

Comment:
Ideally we would also color the items to match inspect, but we can do this separately. For now maybe let's use `#61758A` that we use for regular output?

Reply:
You can look at the names [here](https://github.com/livebook-dev/livebook/blob/42d2386a2d4d3d115946fc6d9a98f2df6319f7a4/lib/livebook/runtime/evaluator/default_formatter.ex#L69-L87) and then their colors [here](https://github.com/livebook-dev/livebook/blob/42d2386a2d4d3d115946fc6d9a98f2df6319f7a4/assets/css/ansi.css) :)